src/commands/SetProjectCommand.cpp

```diff
@@ -88,12 +88,6 @@ bool SetProjectCommand::Apply(const CommandContext & context)
    if( bHasName )
       window.SetLabel(mName);
 
-   if( bHasRate && mRate >= 1 && mRate <= 1000000 )
```

Comment:
I think it is wrong to delete this.  Instead change it to modify the ProjectRate object.

Reply:
I'm sure it's wrong -- the command parameter has no effect now